dashboard/app/controllers/passwords_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
         return
       end
     end
-    unless verify_recaptcha || (current_user && current_user.admin?)
+    unless verify_recaptcha || (current_user&.admin?)
```

Comment:
nit: the parentheses aren't needed anymore

Reply:
Oh, good catches, thanks! I think I know what rule I'm gonna address next

https://github.com/code-dot-org/code-dot-org/blob/58f62036d1e7caecd9410fb245b7a038aa464051/.rubocop_todo.yml#L696-L699

Reply:
Sounds good! Fixing them as a follow up sounds good to me